src/platform/silabs/efr32/BLEChannel.cpp

```diff
@@ -0,0 +1,319 @@
+#include "BLEChannel.h"
+#include <crypto/RandUtils.h>
+#include <lib/support/logging/CHIPLogging.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <platform/internal/CHIPDeviceLayerInternal.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+namespace {
+
+// Side Channel UUIDS
+const uint8_t kSideServiceUUID[16] = { 0x01, 0x00, 0x00, 0xEE, 0xFF, 0xC0, 0x00, 0x80,
```

Comment:
how are those uuid decide? Could a user want to set those?
Maybe we can use defines for them and use
ifndef
define

Reply:
A user would implement their own BLEChannelImpl.cpp object. I just realized this should instead be named:
BLEChannelSideImpl.cpp or something like that.